conda_smithy/lint_recipe.py

```diff
@@ -36,7 +35,12 @@ def lintify(meta):
 
     # 4: The recipe should have some tests.
     if 'test' not in major_sections:
-        lints.append('The recipe must have some tests.')
+        test_files = ['run_test.py', 'run_test.sh', 'run_test.bat', 'run_test.pl']
```

Comment:
Nope, there is no ``run_test.R``: https://github.com/conda/conda-build/blob/62894e75be02e4d048d6180ce9de250a46cd37fa/conda_build/create_test.py